internal/report/report_test.go

```diff
@@ -17,10 +17,12 @@ func TestGetPeriodStart(t *testing.T) {
 
 	tests := map[string]struct {
 		period int
+		time   int64
 
 		wantErr error
 	}{
-		"Valid Period": {period: 500},
+		"Valid Period":   {period: 500, time: 100000},
+		"Negative Time:": {period: 500, time: -100000},
```

Comment:
I think a test case with a time that isn't a multiple of the period would make this complete